packages/manager/apps/dedicated/client/app/app.module.js

```diff
@@ -66,11 +66,6 @@ import './account/otrs/otrs-controllers.module';
 import './account/otrs/otrs-directives.module';
 import './account/otrs/otrs-filters.module';
 import './account/otrs/otrs-services.module';
```

Comment:
Those are not migrated ?

Reply:
`Otrs` and `contacts` are not migrated yet.

They dont come under same url `'/useraccount/...'`

I might have to migrate `contacts` though, before i can merge this, as it is in the same `UserAccount` Module